docs/langref/index.rst

```diff
@@ -1,10 +1,43 @@
 Language Reference
 ==================
-This document provide references to
-embedded languages in TVM stack.
+This document provides references to
+embedded languages in the TVM stack.
+
+Hybrid Script
+-------------
+
+The below page describes the TVM hybrid script front-end,
+which uses software emulation to support some constructs not
+officially supported in TVM.
 
 .. toctree::
    :maxdepth: 2
 
-   relay_op
    hybrid_script
+
+
+
+Introduction to Relay
```

Comment:
consider move Relay before Hybrid Script, as relay is more user-facing

Reply:
Good point, I'll do that.